app/views/books/_book.html.erb

```diff
@@ -1,6 +1,7 @@
 <tr>
   <td class="px-3"><%= book.name %></td>
   <td class="px-3"><%= book.isbn %></td>
+  <td><img class="m-2 img-thumbnail" width="150" src="<%= url_for(book.cover) rescue nil %>" /></td>
```

Comment:
I have a few recommendations here.

1. No need to attempt to show the image if we know there is none:

```erb
  <td>
    <% if book.cover.attached? %>
      <img class="m-2 img-thumbnail" width="150" src="<%= url_for(book.cover) %>" />
    <% end %>
  </td>
```

2. Use Rails view helpers where possible, to allow Rails to handle some of the lower level details. In this case we have `image_tag` at our disposal:

```erb
  <td>
    <% if book.cover.attached? %>
      <%= image_tag(book.cover, class: "m-2 img-thumbnail", width: "150") %>
    <% end %>
  </td>
```

Notice that we no longer need to generate a url for the image, we can simply pass `book.cover` into the `image_tag`.